redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -17,6 +17,7 @@ class rex_mailer extends PHPMailer
     public function __construct($exceptions = false)
     {
         $addon = rex_addon::get('phpmailer');
+        $this->Timeout = 30;
```

Comment:
ich denke wir sollten hier mit 10 sekunden o.ä. arbeiten.

in vielen servern wird die 30 auch schon länger sein als die max execution time

Reply:
Ok.

Reply:
30 Sekunden sind meines Wissens bspw XAMPP-Standardeinstellung - bei einem anderen Hoster gerade nachgeschaut, mit dem ich hauptsächlich arbeite: Da sind es auch 30 Sekunden.

Reply:
Dann sind 10 Sekunden gut.  Wenn ein Server nicht sofort antwortet .. ist er meist eh tot